backend/src/tasks/test/notifications.test.ts

```diff
@@ -0,0 +1,223 @@
+nodemailer.createTransport.mockReturnValue({ sendMail: sendMailMock });
+
+beforeEach(() => {
+  sendMailMock.mockClear();
+  nodemailer.createTransport.mockClear();
+});
+
+describe('notifications', () => {
+  let organization;
+  let organization2;
+  let organization3;
+  beforeAll(async () => {
+    await connectToDatabase();
+    organization = await Organization.create({
+      name: 'CISA',
+      rootDomains: ['test-' + Math.random()],
+      ipBlocks: [],
+      isPassive: false
+    }).save();
+
+    organization2 = await Organization.create({
+      name: 'test-' + Math.random(),
+      rootDomains: ['test-' + Math.random()],
+      ipBlocks: [],
+      isPassive: false
+    }).save();
+
+    organization3 = await Organization.create({
+      name: 'test-' + Math.random(),
+      rootDomains: ['test-' + Math.random()],
+      ipBlocks: [],
+      isPassive: false
+    }).save();
+  });
+  test('list all organizations', async () => {
+    const response = await request(app)
+      .get(`/organizations`)
+      .set(
+        'Authorization',
+        createUserToken({
+          userType: UserType.GLOBAL_ADMIN
+        })
+      )
+      .expect(200);
+    await notifications(
+      {
+        organizations: organization,
+        organization2,
+        organization3
+      },
+      {} as any,
+      () => void 0
+    );
+    expect(response.body.length).toBeGreaterThan(2);
+  });
+
+  test('Get org admins ', async () => {
+    const firstName = 'first names';
+    const lastName = 'last names';
+    const email = Math.random() + '@crossfeed.cisa.gov';
+    const user = await User.create({
+      firstName: '',
+      lastName: '',
+      email: Math.random() + '@crossfeed.cisa.gov'
+    }).save();
+    const user2 = await User.create({
+      firstName: '',
+      lastName: '',
+      email: Math.random() + '@crossfeed.cisa.gov'
+    }).save();
+    const user3 = await User.create({
+      firstName: '',
+      lastName: '',
+      email: Math.random() + '@crossfeed.cisa.gov'
+    }).save();
+    await Role.create({
+      role: 'admin',
+      approved: false,
+      organization: organization,
+      user: user
+    }).save();
+    await Role.create({
+      role: 'admin',
+      approved: false,
+      organization: organization2,
+      user: user2
+    }).save();
+    await Role.create({
+      role: 'user',
+      approved: false,
+      organization: organization3,
+      user: user3
+    }).save();
+
+    const response = await request(app)
+      .post(`/users`)
+      .set(
+        'Authorization',
+        createUserToken({
+          roles: [{ org: organization.id, role: 'admin' }]
+        })
+      )
+      .send({
+        firstName,
+        lastName,
+        email,
+        organization: organization.id,
+        organizationAdmin: true
+      })
+      .expect(200);
+    expect(response.body.roles[0].role).toEqual('admin');
+  });
+
+  test('getting reports list for all organizations', async () => {
```

Comment:
Do we need this test? this isn't about notifications

Reply:
removed in [967bf0e](https://github.com/cisagov/crossfeed/pull/1709/commits/967bf0e8bd390b5560e8381d590f64a4ec528a16)